pipelines/managed/rh-push-to-registry-redhat-io/rh-push-to-registry-redhat-io.yaml

```diff
@@ -62,6 +62,23 @@ spec:
     - name: taskGitRevision
       type: string
       description: The revision in the taskGitUrl repo to be used
+    - name: ociStorage
```

Comment:
these params all need to be added to the README

Reply:
fixed